graphql/models/Article.graphql

```diff
@@ -0,0 +1,30 @@
+
+type Query {
```

Comment:
this will override the `Query` type, you need to use `extend`  https://github.com/salamaashoush/informme/pull/6/files#r709028867